File: codegen/results/v002/qwen3.5_27b-nvfp4_think/20260412_035034/python/003_fibonacci/solution.py
import sys

if __name__ == "__main__":
    try:
        # Attempt to parse the integer argument from command line
        n = int(sys.argv[1])
    except (IndexError, ValueError):
        # If argument is missing or not an integer, do nothing
        pass
    else:
        # Initialize Fibonacci sequence starting with 1, 1
        a, b = 1, 1
        
        # Print numbers in the sequence that do not exceed n
        while a <= n:
            print(a)
            a, b = b, a + b

File: codegen/results/v002/qwen3.5_27b-nvfp4_think/20260412_035034/typescript/003_fibonacci/solution.ts
const input: string | undefined = process.argv[2];
const N: number = parseInt(input || "", 10);

// If N is not a valid number or less than 1, print nothing and exit
if (isNaN(N) || N < 1) {
    process.exit(0);
}

let a: number = 0;
let b: number = 1;

// Generate Fibonacci sequence starting from 1, 1, 2...
while (b <= N) {
    console.log(b);
    [a, b] = [b, a + b];
}